crd/fleet/fleet-crds.yaml

```diff
@@ -3975,6 +5558,11 @@ spec:
                 x-kubernetes-list-map-keys:
                 - type
                 x-kubernetes-list-type: map
+              createTime:
```

Comment:
Replace "the time that the internal failover" with "the time when the internal failover"